for_lakshmi.md

```diff
@@ -0,0 +1,28 @@
+[!] `st2ctl status` shall not have root privileges: st2ctl status, st2ctl reload are things that a non-admin user may likely need.
+
+[!] st2client needs to be available as a separate package ASAP. When we install, shall we do apt-get install st2 st2mistral st2client, or make it part of bundle?
```

Comment:
Yeah, when you say ASAP, you mean once we ship the way we have right now, right?